test/behave/skill-alarm.feature

```diff
@@ -494,3 +448,40 @@ Feature: Alarm skill functionality
     | remove all alarms |
     | remove every alarm |
     | delete every alarm |
+
+# Jira MS-72 https://mycroft.atlassian.net/browse/MS-72
+  Scenario Outline: user snoozes a beeping alarm
```

Comment:
These significantly increase the time it takes for these tests to run. So before this gets merged into the marketplace we really need to think through which tests are going to run when, and how we define that.

Reply:
Yes, I considered adding configurable ability for snooze so we could drop it to like 1 minute during testing but there are only so many hours in a day. I am not adverse to removing the snooze tests or any other tests causing excessive execution time but losing tests does not feel right.